fission/spec.go

```diff
@@ -441,6 +441,7 @@ func specApply(c *cli.Context) error {
 		}
 
 		if !watchResources {
+			pkgWatchCancel()
```

Comment:
Won't this crash in the case that (watchResources==false && waitForBuild==false)?

Reply:
You're right, didn't notice this problem. Fix in #483

Reply:
I moved L431 out of if condition, it's more clean and easy to understand.
https://github.com/fission/fission/blob/fe9453a92eec70ff63dd38166da84e7d85c7381e/fission/spec.go#L429-L433